features/step_definitions/logging.rb

```diff
@@ -1056,3 +1056,125 @@
     | pod_label       | #{pod_label}     |
   })
 end
+
+
+
+Given /^I have(?: "(\w+)")? log pod in project #{QUOTED}$/ do | log_type, project_name |
+    log_type ||= "any"
+    file_dir = "#{BushSlicer::HOME}/testdata/logging/loggen"
+
+    unless project(project_name).exists?
+        step %Q/I run the :new_project client command with:/,table(%{
+          | project_name | #{project_name} |
+        })
+        raise "Error creating namespace" unless @result[:success]
+    end
+
+    case log_type
+      when "json"
+        template_file = "#{file_dir}/container_json_log_template.json"
+      when "flat"
+        template_file = "#{file_dir}/container_json_unicode_log_template.json"
+      when "unicode"
+        template_file = "#{file_dir}/container_json_unicode_log_template.json"
+      else
+        template_file = "#{file_dir}/container_json_log_template.json"
+      end
+
+    unless  project.pods(by:user).size > 0 
+        step %Q/I run the :new_app client command with:/,table(%{
+           | file | #{template_file} |
+         })
```

Comment:
@anpingli Do we need to check the pod status here?

Reply:
To be more accurate, we need to check if the pod is printing logs. How about keep it ASIS and update it in future.

Reply:
Please update here.